src/advanced/node-management.yaml

```diff
@@ -0,0 +1,361 @@
+          description: JSON boolean with current consensus state.
+          content:
+            application/json:
+              schema:
+                type: boolean
+                example: true
+    post:
+      tags: [Private API]
+      summary: Enable/disable consensus interaction
+      description: >
+        Switches consensus interaction of the node on or off.
+      requestBody:
+        required: true
+        description: JSON object indicating the desired consensus state.
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                enabled:
+                  type: boolean
+              example:
+                enabled: false
+      responses:
+        200:
+          description: Successfully changed consensus state.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/NullObject"
+
+  /network:
+    get:
+      tags: [Private API]
+      summary: Get network information
+      description: >
+        Get information about the serialization protocol
+        and the services functioning on the network.
+      responses:
+        200:
+          description: JSON object with network information.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  core_version:
+                    type: string
+                    nullable: true
+                    description: Version of the `exonum` crate.
+                  protocol_version:
+                    type: integer
+                    description: Major version of the Exonum serialization protocol.
+                  services:
+                    type: array
+                    description: Information about services functioning on the network.
+                    items:
+                      $ref: "#/components/schemas/ServiceInfo"
+                example:
+                  core_version: "0.9.0"
+                  protocol_version: 0
+                  services:
+                    - id: 128
+                      name: "cryptocurrency"
+
+  /shutdown:
+    post:
+      tags: [Private API]
+      summary: Shutdown node
+      description: >
+        After receiving a shutdown message, the node stops processing transactions,
+        participating in consensus and terminates after all messages in the event queue
+        are processed.
+      requestBody:
+        required: true
+        description: >
+          `null` JSON object.
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/NullObject"
+      responses:
+        200:
+          description: Successfully initiated node shutdown.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/NullObject"
+
+components:
+  schemas:
+    NullObject:
```

Comment:
Looks clunky, but I'm not sure if a better type description is possible (e.g., `type: "null"` doesn't seem to work).

Reply:
Yeah. For some reason OpenAPI does not have `type: null` and this seems to be the closest we can get to 'only `null` is allowed here'. Though, technically this type specification allows any object in addition to `null`.